index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Portfolia</title>
+    <title>Portfolio website</title>
```

Comment:
- [ ] Kindly consider making sure your page title is adequate and it briefly describes the content of the page. For example:
```
Poor titles:

Microverse project | Your portfolio.
Portifolio project


Better page titles:

Mark's portifolio
Mark full-stack webdeveloper
```
to make it more informative